src/main/java/com/study/gamereviewrest/exceptionHangler/GameNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package com.study.gamereviewrest.exceptionHangler;
+
+public class GameNotFoundException extends RuntimeException {
+    public GameNotFoundException(String message) {
```

Comment:
why is this a runtime exception, read the difference between runtime exception and normal exception